src/include/miopen/tensor.hpp

```diff
@@ -251,6 +256,9 @@ struct TensorDescriptor : miopenTensorDescriptor
 
     miopenDataType_t type             = miopenFloat;
     miopenTensorLayout_t tensorLayout = miopenTensorNCHW;
+
+    std::vector<double> quantScales{std::vector<double>(1, 1.0f)};
+    std::vector<double> quantBiases{std::vector<double>(1, 0.0f)};
```

Comment:
I have some doubts on whether we want to put the quantization states in such a generic place at `tensor.hpp`. The current design seem to indicate the following:
 - We can only do uniform quantization
 - Quantization will and only be able to carry on a single axis
 - That axis is "implicitly" set as the channel axis
Are we okay with all of those assumptions?

Also, the title seems to be pointing at supporting int8 quantization only. If so I think we need some asserts on:
 - This tensor to be int8 data type
 - The size of quantization scale and bias equivalent with channel size
 - They must both exist or both empty. MIOpen shouldn't allow empty scale and a filled bias

Lastly, what does the framework plan to do with those APIs? Can I request their unit test examples?

Reply:
@jerryyin  Hi Zhuoyan, thanks for your valuable comments. I set up a meeting with you, Jun, Jing, Jehandad to discuss about these concerns.

Reply:
@iq136boy Sure, we can discuss about those, but please also include MIGraphX side people. I myself is not all clear about the implementation details they are going to adopt now. If you are to implement the MIOpen/CK implementation, it is best that you also talk with @turneram who work with MIGraphX/CK implementation on this as well.

Reply:
Besides, could you also follow-up closely with @atamazov's suggestions? He is much more experienced than me, and it is important that you address and engage with his comments.

Reply:
@iq136boy After you have offline discussions, please make sure to sync up your conclusions/confusions to this review. This way the reviewers will have an idea what you have talked about.

Also, don't hesitate to reach out to me if you want to clarify things. This is a interactive process. For myself personally, it only become clear to me what the requirements are until I sync up with stakeholders.

Reply:
@jerryyin @iq136boy Please look at #2045 and at https://github.com/ROCmSoftwarePlatform/MIOpen/pull/1625#issuecomment-1483457193.